irr/src/os.cpp

```diff
@@ -160,7 +160,7 @@ void Printer::print(const c8 *message, ELOG_LEVEL ll)
 
 	// Android logcat restricts log-output and cuts the rest of the message away. But we want it all.
 	// On my device max-len is 1023 (+ 0 byte). Some websites claim a limit of 4096 so maybe different numbers on different devices.
-	const size_t maxLogLen = 1023;
+	const int maxLogLen = 1023;
```

Comment:
Maybe make this line constexpr

Reply:
Doesn't really matter but sure, why not.